main/libavif/template.py

```diff
@@ -1,35 +1,34 @@
 pkgname = "libavif"
 pkgver = "1.0.4"
-pkgrel = 0
+pkgrel = 1
 build_style = "cmake"
 configure_args = [
     "-DAVIF_BUILD_APPS=ON",
     "-DAVIF_BUILD_GDK_PIXBUF=ON",
     "-DAVIF_CODEC_DAV1D=ON",
     "-DAVIF_CODEC_AOM=ON",
-    "-DAVIF_BUILD_TESTS=OFF",
-    "-DAVIF_ENABLE_WERROR=OFF",
-    "-DAVIF_ENABLE_GTEST=OFF",
+    "-DAVIF_BUILD_TESTS=ON",
+    "-DAVIF_ENABLE_GTEST=ON",
 ]
-make_check_target = "avif_test_all"
-hostmakedepends = ["cmake", "ninja", "pkgconf", "gdk-pixbuf-devel"]
+hostmakedepends = ["cmake", "ninja", "pkgconf"]
 makedepends = [
-    "gdk-pixbuf-devel",
     "dav1d-devel",
+    "gdk-pixbuf-devel",
+    "gtest-devel",
     "libaom-devel",
     "libjpeg-turbo-devel",
     "libpng-devel",
+    "libwebp-devel",  # for libsharpyuv
```

Comment:
i also thought of splitting a libsharpyuv subpackage from libwebp, what do you think?

Reply:
i don't see the point (this is pulled in every deptree ever)

don't need the comment either since it's just a webp lib and comes from it so it's not special